src/components/ResultsMap.vue

```diff
@@ -12,9 +12,14 @@
 				:position="m.geometry.location"
 				:clickable="true"
 				:draggable="false"
-				@click="handleMarkerClick(m.name)"
+				@click="handleMarkerClick($event, m)"
+			/>
+			<GmapMarker v-if="this.selectedMarker"
```

Comment:
This is the new marker that overlays on top of an existing marker if `selectedMarker` is truth. Its position changes with `selectedMarker`'s location data

Reply:
Thanks, that makes sense!

Reply:
That's a very interesting approach to it! While this vue2-google-maps has helped a lot in some ways, we have definitely run into some of its shortcomings with more complex stuff. Nice solution!